types/clusterize.js/index.d.ts

```diff
@@ -17,19 +18,33 @@ declare class Clusterize {
 }
 
 declare namespace Clusterize {
-    interface Options {
-        scrollId: string;
-        contentId: string;
-        rows?: string[];
-        tag?: string;
-        rows_in_block?: number;
-        blocks_in_cluster?: number;
-        show_no_data_row?: boolean;
-        no_data_text?: string;
-        no_data_class?: string;
-        keep_parity?: boolean;
-        callbacks?: Callbacks;
-    }
+    type Options =
+        | {
+              scrollId: string;
+              contentId: string;
+              rows?: string[];
+              tag?: string;
+              rows_in_block?: number;
+              blocks_in_cluster?: number;
+              show_no_data_row?: boolean;
+              no_data_text?: string;
+              no_data_class?: string;
+              keep_parity?: boolean;
+              callbacks?: Callbacks;
+          }
+        | {
+              scrollElem: HTMLElement;
+              contentElem: HTMLElement;
+              rows?: string[];
+              tag?: string;
+              rows_in_block?: number;
+              blocks_in_cluster?: number;
+              show_no_data_row?: boolean;
+              no_data_text?: string;
+              no_data_class?: string;
+              keep_parity?: boolean;
+              callbacks?: Callbacks;
+          };
```

Comment:
Nice work, you probably want something more like:

```
type Setup = {  scrollId: string; contentId: string; } | {  scrollElem: HTMLElement | contentElem: HTMLElement }   type Options = Setup & { rows?: string[], ... }
```

To avoid the duplication